src/brasil/gov/portal/browser/content/templates/infographic.pt

```diff
@@ -0,0 +1,29 @@
+<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en"
+    xmlns:tal="http://xml.zope.org/namespaces/tal"
+    xmlns:metal="http://xml.zope.org/namespaces/metal"
+    xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+    lang="en"
+    metal:use-macro="context/main_template/macros/master"
+    i18n:domain="plone">
+<body>
+
+<metal:content-core fill-slot="content-core">
+<metal:block define-macro="content-core" tal:define="size context/image/getSize">
+  <div class="image-product">
+    <a class="discreet"
+        tal:attributes="href string:${context/@@plone_context_state/object_url}/image_view_fullscreen"
+        tal:define="scale context/@@images;
+                    img_tag python:scale.scale('image', scale='large').tag()"
+        tal:on-error="string: Image cannot be displayed">
+    <img tal:replace="structure img_tag" />
```

Comment:
could you please fix the indentation of this file?

Reply:
it's a copy of the image template.